src/models/neural_networks/ealstm.py

```diff
@@ -297,8 +301,19 @@ def forward(
             static_x.append(static)
         if self.include_pred_month:
             static_x.append(pred_month)
+
+        # append prev_y to DYNAMIC data
```

Comment:
the previous y value is not static so should be appended to the dynamic data instead of the static data